ChatQnA/README.md

```diff
@@ -97,6 +97,123 @@ flowchart LR
 
 This ChatQnA use case performs RAG using LangChain, Redis VectorDB and Text Generation Inference on Intel Gaudi2 or Intel XEON Scalable Processors. The Intel Gaudi2 accelerator supports both training and inference for deep learning models in particular for LLMs. Visit [Habana AI products](https://habana.ai/products) for more details.
 
+For a full list of framework, model, serving, and hardware choices available for each of the microservice components in the ChatQnA architecture, please refer to the below table:
+
```

Comment:
```suggestion
<details>
<summary> Gaudi default compose.yaml </summary>
| MicroService    | Open Source Project                |   HW      |  Port     | Endpoint |
| --------------- | ---------------------------------- |---------- |
| Embedding     | Langchain                                  | Gaudi     | 6000 |  /v1/embaddings |
| Retriever         | Langchain                                  | Xeon      | 7000 |  /v1/retrieval         |
| Reranking       | Langchain                                  | Xeon      | 8000 | /v1/reranking        |
| LLM                |  Langchain                                  | Gaudi     | 9000 | /v1/chat/completions |
| Dataprep        |  Redis                                          | Xeon      | 6007| /v1/dataprep             |

</details>
```

Reply:
Thank you. Revised according to comments. Included simple table in latest commit.